odoo/addons/base/models/res_lang.py

```diff
@@ -46,7 +48,9 @@ class Lang(models.Model):
              "[1,2,-1] will represent it to be 106,50,0;[3] will represent it as 106,500. "
              "Provided ',' as the thousand separator in each case.")
     decimal_point = fields.Char(string='Decimal Separator', required=True, default='.', trim=False)
+    decimal_point_preview = fields.Char(string="Decimal separator preview", compute="_compute_decimal_point_preview")
```

Comment:
We usually capitalize everything in labels, so "Decimal Separator Preview".

(to apply to other new fields).